docs/modules/configuration/pages/secrets-management.adoc

```diff
@@ -151,6 +151,10 @@ Token authentication is just one of several authentication methods that are http
 |Vault Enterprise allows using https://developer.hashicorp.com/vault/docs/enterprise/namespaces[namespaces] to isolate multiple Vaults on a single Vault server. This feature is not supported by Vault Community edition and has no effect on Vault operations.
 |`ns1/ns2`
 
+|[subs=+quotes]`*secrets-manager.vault.enabled*`
+|Controls the activation of the processor that resolves property values from Secrets Manager. `true` by default. If set to `false`, values from HashiCorp’s Vault will not be retrieved. Can be used when access to the Secret Manager is restricted, e.g. for local debugging, to override only the required properties instead of all processed ones.
```

Comment:
our users have no idea what 'processor' is, please make sure the docs are clear for end users